client/test/data/test.cto

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
I'd recommend using a deeper test folder structure here to help you out further down the line:
data/valid
data/valid/cto
data/valid/acl
data/valid/project

data/invalid

etc, etc

Otherwise once you build up a large number of resources, it will be unmaintainable

Reply:
I will add these as needed per our chat.

Reply:
move the files to data/valid/cto  and data/valid/acl now and will update others as needed.